atsign-foundation/at_widgets
# Add at_app_flutter repo

**- What I did**

Add at_app_flutter, I needed to separate the flutter sdk dependencies from the at_app executable.
I've moved them to this at_app_flutter package.

**- How I did it**

Separate the necessary files from at_app and create a new package containing them.

**- How to verify it**

Verify by testing it with the at_app create command:

```sh
flutter pub global deactivate at_app
flutter pub global activate at_app
at_app create path/to/test/folder
```

**- Description for the changelog**
Add at_app_flutter package.
